### ISSUE TITLE
Issue when scraping 1pondo, caribbeancom, caribbeancompr

### ISSUE THREAD

**Issue by mut3k1** [Wizell/JAVMovieScraper#2]
First of all, thank you so much for forking this project!  I wish I had the ability to update it myself.

One thing that has been broken for a while is pulling from both 1pondo, and caribbeancom premimum.

For 1pondo and Caribbeancom it's stopped scraping images and meta information.  1 pondo will scrape a 404 not found meta and no image whilst Caribbeancom will pull down meta info but no image anymore.

For Caribbeancom Premium, it's been broken for some time but if you provide a manual URL it works perfectly!

Thanks,

Mut3ki1

**Comment by Wizell** [comment]
Please avoid putting more than one issue per issue.

For 1pondo i fixed the 404 message but there is still many issues that may need some time to sort out.

For caribbeancom premimum i don't see issues so far. Can you please at least provide an example.

**Comment by DarkMask100** [comment]
caribbeancom info is pulled is pulled when I select caribbeancom as the scraper. However no images for the video or actors images are being pulled.

**Comment by mut3k1** [comment]
The bottom is output for Caribbeancom, as DarkMask100 says, it pulls data but no images
![image](https://user-images.githubusercontent.com/32484013/31500116-96ab76ee-afa1-11e7-9419-af5f86ca4afd.png)

CaribPR doesn't pull anything for me - perhaps my files are named wrong?

![image](https://user-images.githubusercontent.com/32484013/31500237-dbe677c2-afa1-11e7-9db8-0fd315d76425.png)

Thanks for fixing the 404 issue in 1pondo, here's the output now (no images).

![image](https://user-images.githubusercontent.com/32484013/31500296-0418f7c4-afa2-11e7-8f42-81a8849c2532.png)

**Comment by Wizell** [comment]
Caribbeancom webpages does not have actresses or poster so it cannot be scraped.

For Caribbeancom premium i do have arts. I just tried with id 101417-519. 

For 1pondo there is a bug. I'm looking into that

**Comment by DarkMask100** [comment]
I've noticed that with Caribbeancom files, I can get the images from JavBus but since the Caribbeancom scraper is separate, I don't get the info. 

This issue could be solved by either having a separate amalgamated group for uncensored or allowing us to add scrapers to the group.

**Comment by Wizell** [comment]
Correct me if i didn't got what you meant properly.
From what i got, you mean to scrap some data with carribeancom and images from javbus ?

About the amalgamation, in the current state a huge and horrible mess, i'm rewritting the whole stuff to allow custom arbirary groups with easier configuration window and configuration file. But this work is quite huge and will take quite some time.

**Comment by DarkMask100** [comment]
Correct, JavBus scraper provides images but no data while Caribbeancom scraper provides the data and currently there is no way to use both to obtain the best results.
